volume.go

```diff
@@ -153,13 +186,29 @@ func (v *volume) Attachments() []VolumeAttachment {
 	return result
 }
 
+// AttachmentPlans implements Volume.
+func (v *volume) AttachmentPlans() []VolumeAttachmentPlan {
+	var result []VolumeAttachmentPlan
+	for _, attachment := range v.AttachmentPlans_ {
+		result = append(result, attachment)
+	}
+	return result
+}
+
 // AddAttachment implements Volume.
 func (v *volume) AddAttachment(args VolumeAttachmentArgs) VolumeAttachment {
 	a := newVolumeAttachment(args)
 	v.Attachments_.Attachments_ = append(v.Attachments_.Attachments_, a)
 	return a
 }
 
+// AddAttachment implements Volume.
+func (v *volume) AddAttachmentPlan(args VolumeAttachmentPlanArgs) VolumeAttachmentPlan {
```

Comment:
I know it is trivial code, but I think we should add a test for this as per `TestAddingAttachments`.

Reply:
Adding now